Aquality.Selenium.Core/src/Aquality.Selenium.Core/Visualization/VisualStateProvider.cs

```diff
@@ -27,19 +27,31 @@ public VisualStateProvider(IImageComparator imageComparator, IElementActionRetri
         public Point Location => GetLoggedValue(nameof(Location), element => element.Location);
 
         public Image Image 
-            => GetLoggedValue(nameof(Image), element => element.GetScreenshot().AsImage(), image => image.Size.ToString());
+            => GetLoggedValue(nameof(Image), element => element.GetScreenshot().AsImage(), image => image?.Size.ToString());
 
-        private T GetLoggedValue<T>(string name, Func<RemoteWebElement, T> getValue, Func<T, string> toString = null)
+        private T GetLoggedValue<T>(string name, Func<WebElement, T> getValue, Func<T, string> toString = null)
         {
             logVisualState($"loc.el.visual.get{name.ToLower()}");
-            var value = actionRetrier.DoWithRetry(() => getValue(getElement()));
+            var value = actionRetrier.DoWithRetry(() =>
+                {
+                    try
+                    {
+                        return getValue(getElement());
+                    }
+                    catch (WebDriverException)
+                    {
+                        return default;
```

Comment:
God hope this language construction would work on .NET Standard 2.0 :)

Reply:
discussed. Added necessary changes